docs/aligned-execution.md

```diff
@@ -0,0 +1,5 @@
+Aligned execution means every node applies the same transactions in the same order at the same logical time, requiring no extra synchronization.
```

Comment:
If `Aligned execution` is the new term you're suggesting, then a) I like it, and b) let's defined it bolder!